### PR TITLE
Hardening/1368 style headerguards

### PR DESCRIPTION
**Pull request by kzangeli**
Two style guide rules in all broker code:

* Header Guards
* Lines Ending In Whitespace

### CODE REVIEW

File: test/unittests/common/commonFormat_test.cpp [hunk 1]
```diff
@@ -31,15 +31,15 @@
```
Comment (fgalan) [inline test/unittests/common/commonFormat_test.cpp]: I thougth that all the test/unittests/ directories were covered... However, this PR include fixes in one of them. Why the existing checks have not detected this? Maybe the directory has not been included in the style check script list?

File: test/unittests/convenience/AppendContextElementRequest_test.cpp [hunk 2]
```diff
@@ -37,7 +37,7 @@
```
Comment (fgalan) [inline test/unittests/convenience/AppendContextElementRequest_test.cpp]: Same here

File: test/unittests/ngsi/AttributeDomainName_test.cpp [hunk 3]
```diff
@@ -33,7 +33,7 @@
```
Comment (fgalan) [inline test/unittests/ngsi/AttributeDomainName_test.cpp]: Same here for /ngsi

File: test/unittests/convenience/AppendContextElementRequest_test.cpp (hunk 2 above)
Comment (fgalan) [inline test/unittests/convenience/AppendContextElementRequest_test.cpp]: ...for /convenience

File: test/unittests/common/commonFormat_test.cpp (hunk 1 above)
Comment (fgalan) [inline test/unittests/common/commonFormat_test.cpp]: ...for /common

File: test/unittests/ngsi10/NotifyContextRequest_test.cpp [hunk 4]
```diff
@@ -38,7 +38,7 @@
```
Comment (fgalan) [inline test/unittests/ngsi10/NotifyContextRequest_test.cpp]: Same here for /ngsi10

File: test/unittests/ngsi9/DiscoverContextAvailabilityRequest_test.cpp [hunk 5]
```diff
@@ -64,7 +64,7 @@
```
Comment (fgalan) [inline test/unittests/ngsi9/DiscoverContextAvailabilityRequest_test.cpp]: Same here for /ngsi9

File: test/unittests/parse/CompoundValueNode_test.cpp [hunk 6]
```diff
@@ -34,7 +34,7 @@
```
Comment (fgalan) [inline test/unittests/parse/CompoundValueNode_test.cpp]: Same here for /parse

File: test/unittests/common/commonFormat_test.cpp (hunk 1 above)
Comment (kzangeli) [inline test/unittests/common/commonFormat_test.cpp]: Ah no, lots of dirs are still missing in unit tests.
See the commented lines in script style_guide_in_makefile.sh

Comment (fgalan) [inline test/unittests/common/commonFormat_test.cpp]: NTC

File: test/unittests/convenience/AppendContextElementRequest_test.cpp (hunk 2 above)
Comment (fgalan) [inline test/unittests/convenience/AppendContextElementRequest_test.cpp]: NTC

File: test/unittests/ngsi10/NotifyContextRequest_test.cpp (hunk 4 above)
Comment (fgalan) [inline test/unittests/ngsi10/NotifyContextRequest_test.cpp]: NTC

File: test/unittests/parse/CompoundValueNode_test.cpp (hunk 6 above)
Comment (fgalan) [inline test/unittests/parse/CompoundValueNode_test.cpp]: NTC

File: test/unittests/ngsi9/DiscoverContextAvailabilityRequest_test.cpp (hunk 5 above)
Comment (fgalan) [inline test/unittests/ngsi9/DiscoverContextAvailabilityRequest_test.cpp]: NTC

File: test/unittests/ngsi/AttributeDomainName_test.cpp (hunk 3 above)
Comment (fgalan) [inline test/unittests/ngsi/AttributeDomainName_test.cpp]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**